walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -45,89 +46,96 @@
       }
     }
 
-    Collections.sort(conflicts, TimeRange.ORDER_BY_START);
-
-    return conflicts;
+    return ImmutableList.sortedCopyOf(
+        TimeRange.ORDER_BY_START.thenComparing(TimeRange.ORDER_BY_END.reversed()), conflicts);
   }
 
-  public List<TimeRange> mergeOverlappingRanges(List<TimeRange> conflicts) {
+  private ImmutableList<TimeRange> mergeOverlappingRanges(List<TimeRange> conflicts) {
```

Comment:
It's good to make the method parameter type immutable too. This communicates to the caller that their input won't be modified.